server.go

```diff
@@ -142,12 +142,16 @@ func (s *Server) Finish(ke3 *message.KE3) error {
 	return nil
 }
 
-// SessionKey returns the session key if the previous calls to Init() and Finish() were
-// successful.
+// SessionKey returns the session key if the previous call to Init() was successful.
```

Comment:
updated to more accurately reflect when the session key is set.

Reply:
Initially, I didn't want to make the session key accessible until the full flow was successful. But since we're now exporting the state it's more accurate, indeed